docker/docker-install
# Add fixed time lock timeout for apt-get command

**- What I did**
Added a new argument to apt-get to avoid locks - I came across one on a Proxmox VM when trying to install docker from a script.

Fixes #366 #416

**- How I did it**
I added the argument ```-o DPkg::Lock::Timeout=60``` to apt-get occurrences, this should be enough to acquire the lock.

**- How to verify it**
Run apt-get update and the get-docker script in a second tab. It should wait for 60 seconds to get the lock.

**- Description for the changelog**
Added ```-o DPkg::Lock::Timeout=60``` argument to apt-get occurrences to wait for the lock to be released instead of failing immediately


**- A picture of a cute animal (not mandatory but encouraged)**
My cat (her name is Ruby)
![ruby](https://github.com/docker/docker-install/assets/22376195/52685157-a484-4f5a-a501-e7b0a0f2911c)
